docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -918,6 +919,8 @@ testcafe chrome test.ts --compiler-options typescript.customCompilerModulePath=.
 
 > TestCafe resolves relative paths against its own installation folder.
 
+The `typescript.options.lib` option looks for files in the compiler’s `node_modules/typescript/lib` folder. Include the *full* name of the library file (for example: `lib.webworker.d.ts`) to enable that library during compilation.
```

Comment:
> The `typescript.options.lib` option looks for files in the compiler’s `node_modules/typescript/lib` folder

It's not true. Typescript compiler requires such parameter format for this `lib` option. TestCafe takes the user input and pass it to the compiler. To avoid errors in the parameter format, it is convenient to navigate in `node_modules/typescript/lib` folder and see the correct file names.